ldk/javascript/examples/consolidate-loop/README.md

```diff
@@ -0,0 +1,35 @@
+# Building and running Loop in Olive Helps
+
+### What is consolidate loop?
+
+Consolidate loop demonstrates the capabilities of Olive Helps used in conjunction with each other.
+
+### When to use consolidate loop?
+
+New patient appointment scheduling as a 2-step process: 1) Patient calls, scheduling staff schedules appointment. 2) Registration staff uses this information to contact patients before their appointments to confirm the appointment and provide instructions.
+
+### How to use consolidate loop?
+
+1. Hit Ctrl + N to open Consolidate Loop - Form Whisper
+2. Input patient information and submit.
+   Please note that firstname, lastname, telephone number, email and appointmentDate are required fields.
+3. Use firstname, lastname, date of birth, phone number or appointment date to search for patients using search bar, choose the right one if multiple records are returned.
```

Comment:
Should either be the field names (like "firstName", "phoneNumber", etc.) or the label for the field displayed in the UI ("First Name", "Telephone", etc.)